src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,145 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int GROW_FACTOR = 2;
+
+    private Node<K, V>[] table;
+    private double loadFactor;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        this(DEFAULT_CAPACITY, DEFAULT_LOAD_FACTOR);
+    }
+
+    public MyHashMap(int initialCapacity, double loadFactor) {
```

Comment:
you can use just one constructor instead two and constructor in that task have one purpose to initialize "table"
```suggestion
public MyHashMap() {
        initialArray = new Node[DEFAULT_INITIAL_CAPACITY];
    }
```